feature-libs/quote/components/quote-seller-edit/quote-seller-edit.component.ts

```diff
@@ -70,11 +72,12 @@ export class QuoteSellerEditComponent implements OnInit, OnDestroy {
               )
             );
           this.form.controls.discount.addValidators([numberFormatValidator]);
-          this.form.controls.discount.setValue(
-            localizationElements.formatter.format(
-              quote.quoteDiscounts?.value ?? 0
-            )
-          );
+          const discountValue = quote.quoteDiscounts?.value ?? 0;
```

Comment:
```suggestion
          const discountValue = quote.quoteDiscounts?.value;
```
Is this default still required? - tests are still green if i delete it.

Reply:
Ah yes you are right. Forgot to delete that after my changes